deploy/roles/stattransfer14/defaults/main.yaml

```diff
@@ -0,0 +1,13 @@
+---
+stattransfer_install_directory: /usr/local/StatTransfer14
+#stattransfer_local_bin_directory: /artifacts
+stattransfer_local_bin_directory: /opt/ancillary_local
+stattransfer_license_type: "25 User Lease"
+stattransfer_expiration_date: "August 31, 2018"
+stattransfer_reg_code: STAT-TRANSFER-REG
+stattransfer_auth_code: STAT-TRANSFER-AUTH
+stattransfer_sign_code: STAT-TRANSFER-SIGN
+stattransfer_major_version: "14"
+stattransfer_name: "STAT-TRANSFER-NAME"
+stattransfer_organization: "STAT-TRANSFER-ORGANIZATION"
+stattransfer_set_code: "STAT-TRANSFER-SET-CODE"
```

Comment:
@lel99999 Ideally we won't have a new role for stattransfer14 vs 13, just update the existing role with the new variable values.  That way we can control both versions using the same role while incrementally upgrading machines.